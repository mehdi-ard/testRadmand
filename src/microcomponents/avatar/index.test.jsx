import React from 'react';
import renderer from 'react-test-renderer';
import Avatar from '.';

it('renders default', () => {
  expect(
    renderer
      .create(
        <Avatar src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png" />
      )
      .toJSON()
  ).toMatchSnapshot();
});
it('renders size', () => {
  expect(
    renderer
      .create(
        <Avatar
          size={64}
          src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
        />
      )
      .toJSON()
  ).toMatchSnapshot();
});

it('renders shape square', () => {
  expect(
    renderer
      .create(
        <Avatar
          size="large"
          shape="square"
          src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
        />
      )
      .toJSON()
  ).toMatchSnapshot();
});
