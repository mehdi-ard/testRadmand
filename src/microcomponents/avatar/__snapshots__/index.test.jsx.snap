// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default 1`] = `
<span
  className="ant-avatar ant-avatar-circle ant-avatar-image"
  style={Object {}}
>
  <img
    onError={[Function]}
    src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
  />
</span>
`;

exports[`renders shape square 1`] = `
<span
  className="ant-avatar ant-avatar-lg ant-avatar-square ant-avatar-image"
  style={Object {}}
>
  <img
    onError={[Function]}
    src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
  />
</span>
`;

exports[`renders size 1`] = `
<span
  className="ant-avatar ant-avatar-circle ant-avatar-image"
  style={
    Object {
      "fontSize": 18,
      "height": 64,
      "lineHeight": "64px",
      "width": 64,
    }
  }
>
  <img
    onError={[Function]}
    src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
  />
</span>
`;
